ERROR - 2023-03-12 10:11:04 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\GuestModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\GuestModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-12 10:11:04 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `astayco_myguests`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\GuestModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Guest->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
